apps/alpha-test-app/src/components/layout-demo/MultiSettingsGroup.vue

```diff
@@ -0,0 +1,45 @@
+<script lang="ts" setup>
+import { OnyxCheckboxGroup, type SelectionOption } from "sit-onyx";
+import { computed } from "vue";
+import type { Settings } from "./LayoutSettings.vue";
+
+const activeSettings = defineModel<Settings>();
+
+const props = defineProps<{
+  headline: string;
+  options: SelectionOption<undefined>[];
+  horizontal?: boolean;
+}>();
+
+const options = computed<SelectionOption<Settings>[]>(() =>
+  props.options.map(
+    (option) => ({ ...option, value: { [option.id]: true } }) as SelectionOption<Settings>,
+  ),
+);
+
+const settingsToSelection = (settings?: Settings): (keyof Settings)[] => {
+  return settings ? (Object.keys(settings) as (keyof Settings)[]) : [];
```

Comment:
Shouldn't only true values be included?

Reply:
somehow still works. but technically sounds like it should not ^^ I'll filter it